birdhouse/pavics-compose.sh

```diff
@@ -116,7 +119,7 @@ if [ x"$1" = x"up" ]; then
 fi
 
 # the PROXY_SECURE_PORT is a little trick to make the compose file invalid without the usage of this wrapper script
-PROXY_SECURE_PORT=443 HOSTNAME=${PAVICS_FQDN} docker-compose ${COMPOSE_CONF_LIST} $*
```

Comment:
Should there be some kind of duplicate filtering, or we simply let the command fail with any (possibility ambiguous) error message? 
For example, `COMPOSE_EXTRA_OPTS="-d"` and the user does `pavics-compose up -d` by mistake.

Reply:
Good catch, yes double `-d` do not work !   Weird !  Which reason would motivate such a strict cli args parsing !

Reply:
Often CLI libraries support duplicating flags to increase some level. For example, `-v` means verbose level 1 and `-v -v` means verbose level 2. 
Differentiating between options that support this duplication and those that don't are important to these argument parsing libraries.

Reply:
Right !  But activating this "strict count" for all options seems excessive !